### DISCUSSION TITLE
Idea: Non-intrusive notifications

### DISCUSSION THREAD

**Discussion by teamcons** [elementary/notifications#234] [Ideas]
## Feature

Notifications appear in the top right, at the same place indicator menus shows up

So it is possible during work to need to use an indicator, and have a notification suddenly "block" the action by showing on top of it.
Which requires to click the X, and going back to the applet, adding some frustration at the intrusion and in turn at the DE.

![Bildschirmfoto von 2024-10-05 20 36 34](https://github.com/user-attachments/assets/6f186ead-4bba-4623-91de-c9b4de93caba)

## Idea
**Describe the solution you'd like**

Slight (dock-like for consistency) transparency on hover, allowing to see the items below
And clicks to be registered below instead of on the indicator

Both allowing to continue the action while being informed of something else going on

**Existing work**

The defunct Unity did that, albeit without close button for the notification, and holding up decay while hovered... Which was also frustrating at times

![updates-available](https://github.com/user-attachments/assets/ae02b95f-376a-4b2e-bbbd-5a86baad58d4)

**Describe alternatives you've considered**

Do Not Disturb could be on at all times.
However that means losing notifications alltogether, or forgetting to clear them and getting https://github.com/elementary/wingpanel-indicator-notifications/issues/237

**Additional context**
This may require additional consideration from the community, i am not sure if this annoys only me (and at worst i can live with intrusive bubbles)

**Comment by marmeladapk** [comment]
IMHO indicator menus should push down notifications so that they don't overlap.

**Reply by teamcons** [reply]
Also sounds like a really cool idea (Even better, id say)

**Reply by leolost2605** [reply]
There's an issue about this at elementary/gala#91

**Reply by teamcons** [reply]
> There's an issue about this at [elementary/gala#91](https://github.com/elementary/gala/issues/91)

And a PR on the way !!!! amazing !!! Thank you !!!
